shennah_al-jawad/week_01/Weds-Homework.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<body>
+<script>
+
+//The Word Guesser
+var word = ['D', 'O', 'G'];
+var guesses = ['_', '_', '_',];
+var counter = word.length;
+console.log(guesses);
+console.log("Let the games begin!")
+
+function guessLetter(l) {
```

Comment:
you might want to break this function on few other functions that will do some smaller tasks, and then just call on them inside of guessLetter. PS: just a hint for improvement